react/index.test.ts

```diff
@@ -474,23 +474,17 @@ it('recreating filter on args changes', async () => {
     '3'
   ])
 
+  renders.splice(0, renders.length)
   await act(async () => {
     screen.getByTestId('change').click()
     await delay(10)
   })
   expect(renders).toEqual([
-    'list',
```

Comment:
I think there is no need to test data, that was tested above. Tests will be simpler to understand